arcade/gui/constructs.py

```diff
@@ -25,50 +31,66 @@ def __init__(
         width: float,
         height: float,
         message_text: str,
-        buttons=("Ok",),
-        callback=None
-    ):
+        background: arcade.Texture = arcade.load_texture(
+            ":resources:gui_basic_assets/window/grey_panel.png"
+        ),
+        buttons: Tuple[str, ...] = ("Ok",),
+        disappear: bool = False,
+        disappear_time: float = 3,
```

Comment:
having `disappear` as an argument might be redundant if you have `None` as the default value for `disappear_Time`. I'm not sure about this yet. It might be worth leaving as-is and bringing it up with Eruvanos later.